app/packages/looker-3d/src/action-bar/point-size.tsx

```diff
@@ -1,50 +1,108 @@
 import { PopoutSectionTitle, useTheme } from "@fiftyone/components";
-import { Slider } from "@fiftyone/core/src/components/Common/RangeSlider";
+// import { Slider } from "@fiftyone/core/src/components/Common/RangeSlider";
 import PointSizeIcon from "@mui/icons-material/ScatterPlot";
+import Input from "@mui/material/Input";
+import Slider from "@mui/material/Slider";
+import { useCallback, useMemo, useState } from "react";
 import * as recoil from "recoil";
 import { ActionItem } from "../containers";
 import {
   ACTION_SET_POINT_SIZE,
   currentActionAtom,
   currentPointSizeAtom,
-  pointSizeRangeAtom,
 } from "../state";
 import { ActionPopOver } from "./shared";
+import "./styles.css";
```

Comment:
Please use CSS modules. For mui overrides, I think there are more formal ways to override styles